scripts/SREP/update-upgrade-policy-state/script.sh

```diff
@@ -48,13 +60,23 @@ get_policy_uuid() {
 #   - Policy UUID
 update_upgrade_policy() {
     local ocm_slug="api/clusters_mgmt/v1/clusters/${1}/upgrade_policies/${2}/state"
-    curl -XPATCH \
-        -H "Content-Type: application/json" \
-        -H "Authorization: AccessToken ${CLUSTER_UUID}:${AUTH_TOKEN}" \
-        "https://api.openshift.com/${ocm_slug}" -d '{
-        "value": "cancelled",
-        "description": "Manually cancelled by SRE"
-    }'
+    if [[ -z "$env" ]]; then
```

Comment:
same as I mentioned above (the `if/else` clause contains repeated code), better to have a reusable function.